_dev/js/back/src/core/plugins/segment.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Academic Free License 3.0 (AFL-3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/AFL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/AFL-3.0 Academic Free License 3.0 (AFL-3.0)
+ */
+import Segment from '@prestashopcorp/segment-vue';
+
+export default {
+  install(Vue, { router }) {
+    Vue.use(Segment, {
+      id:
+        process.env.VUE_APP_SEGMENT_API_KEY ||
```

Comment:
I checked the API key on our Segment app, this one hard coded is the good one and I have also the same on my .env file. Don't you have the same, isn't it ?

Reply:
The hardcoded variable will be removed